spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/SparkExecutionPlanTracker.scala

```diff
@@ -66,7 +66,6 @@ class SparkExecutionPlanTracker(
     logWarn(s"Fail to execute query: {$qe}, {$funcName}", exception)
   }
 
-  // TODO: We should consider multiple inputs and multiple outs.
```

Comment:
It seems that we need to add test cases including both `view` and `table` to remove this TODO.

Reply:
I have created a new issue to track this: #17.